### PR TITLE
Refactorin so that language is pulled out of the core and int Drupal7 specific

### PR DESCRIPTION
**Pull request by dutchiexl**

### CODE REVIEW

**Comment by ademarco** [comment]
:+1:

**Comment by dutchiexl** [comment]
Ok I updated thes, I was doubting about it :/

**Comment by ademarco** [comment]
All green on both Drupal 8 and Drupal 7 tests on my local setup, we should push a full rebuild on the drupal-extension side in order to have green light from TravisCI too. @jhedstrom could we do that?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-[![Latest Stable Version](https://poser.pugx.org/drupal/drupal-driver/v/stable.svg)](https://packagist.org/packages/drupal/drupal-driver) [![Total Downloads](https://poser.pugx.org/drupal/drupal-driver/downloads.svg)](https://packagist.org/packages/drupal/drupal-driver) [![License](https://poser.pugx.org/drupal/drupal-driver/license.svg)](https://packagist.org/packages/drupal/drupal-driver) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/jhedstrom/DrupalDriver/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/jhedstrom/DrupalDriver/?branch=master)
+[Read the full documentation](http://drupal-drivers.readthedocs.org)
 
 ### Drivers
```
Comment (jhedstrom) [inline README.md]: Can we back these changes out? Looks like an accidental merge from 1.0 got into the README.

**Comment by jhedstrom** [comment]
Nevermind, I reverted the README change manually.

Thanks for this!

**Comment by dutchiexl** [comment]
Yes, i ma sorry for that

**Pull request merged**